infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -254,7 +254,7 @@ public void setupTopologyConfigId(long topologyConfigId) {
                         } else if (type == LogReplicationMetadataType.VERSION) {
                             // TODO: We should update the version in metadata manager
                             //  when the version is read from static file
-                            String version = LogReplicationConfigManager.getCurrentVersion();
+                            String version = UpgradeManager.getCurrentVersion();
```

Comment:
It seems that this shouldn't be a static as the upgrade manager has already been instantiated and the version is available from the caller. 

I think this code changes radically in the session-based LogReplicationMetadataManager, so I'm fine if we don't address this.